locale/templates/settings.ftl

```diff
@@ -364,7 +370,9 @@ add-secondary-email-enter-address =
 add-secondary-email-cancel-button = Cancel
 add-secondary-email-save-button = Save
 
-##
+## Auth-server based errors that originate from backend service
```

Comment:
Is this comment for a single string or for a section?

Reply:
Hi @peiying2 this string should have been in the same section as the other `auth-error-xxx` strings. 

This seems to be an issue in the code base, I'll make the necessary updates so that this duplicate section is removed and the string is is added to the other `auth-error-xxx` strings.